aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -916,6 +928,20 @@ module aptos_framework::stake {
         let config = staking_config::get();
         let validator_perf = borrow_global_mut<ValidatorPerformance>(@aptos_framework);
 
+        spec {
```

Comment:
These assumptions should be invariants eventually. Currently, Prover cannot prove it due to the lack of loop invariants and some universal quantification issue.